### ISSUE TITLE
linguagem da aplicação 

### ISSUE THREAD

**Issue by joaovitormoitinho** [JoseOda/TrabalhoGCS#16]
Como a maior parte dos usuários vão ser brasileiros, seria interessante trocar a linguagem da aplicação para português do brasil

**Comment by JoseOda** [comment]
Na verdade eu discordo um pouco, trocando a linguagem de aplicação, nosso público ficaria muito nichado para um público mais brasileiro, além disso como tem muitos usuários de git pelo menos, deixar em inglês expandiria o público da nossa aplicação, já que ela é voltada a pesquisa de usuários do github, e não voltado a pesquisa de usuários de github do brasil

**Comment by enzookita** [comment]
@JoseOda concordo com seu ponto de vista, o nosso programa foi criado para ser usado pelo máximo de pessoas, inglês é a principal língua do Ocidente. Acho melhor deixarmos o nosso programa em inglês mesmo.
